### ISSUE TITLE
Accessibility improvements - add "Skip to content" link for header 

### ISSUE THREAD

**Issue by szafran89** [SnowdogApps/magento2-alpaca-components#154]
On the beginning on the header we should add "Skip to content" link. (it's not visible on the website now). it should looks like on Luma theme and our mixin visually-hidden can be used here but visibility on focus should be added.

**Comment by anqaka** [comment]
[dev-51564](https://dev.snowdog.pro/issues/51564)
